index.bs

```diff
@@ -1143,6 +1146,25 @@ The following constraints define when a [=/transaction=] can be
 
 </div>
 
+To <dfn export>cleanup Indexed Database transactions</dfn> for the
+[=event loop=], run the following steps for each [=/transaction=]
+with a matching [=transaction/cleanup event loop=]:
```

Comment:
Hmm, I guess in this case I'd be OK saying your version but with "... matching the current event loop"? It's not really a phrase used elsewhere but it helps make things easier to read here so it works for me.